rcl/include/rcl/time.h

```diff
@@ -487,68 +488,6 @@ rcl_ret_t
 rcl_set_ros_time_override(rcl_time_source_t * time_source,
   rcl_time_point_value_t time_value);
 
-/// Retrieve the current time as a rcl_time_point_value_t.
-/**
- * This function returns the time from a system clock.
- * The closest equivalent would be to std::chrono::system_clock::now();
- *
- * The resolution (e.g. nanoseconds vs microseconds) is not guaranteed.
- *
- * The now argument must point to an allocated rcl_system_time_point_t struct,
- * as the result is copied into this variable.
- *
- * <hr>
- * Attribute          | Adherence
- * ------------------ | -------------
- * Allocates Memory   | No
- * Thread-Safe        | Yes
- * Uses Atomics       | No
- * Lock-Free          | Yes [1]
- * <i>[1] if `atomic_is_lock_free()` returns true for `atomic_int_least64_t`</i>
- *
- * \todo TODO(tfoote): consider moving this to rmw for more reuse
- *
- * \param[out] now a datafield in which the current time is stored
- * \return `RCL_RET_OK` if the current time was successfully obtained, or
- * \return `RCL_RET_INVALID_ARGUMENT` if any arguments are invalid, or
- * \return `RCL_RET_ERROR` an unspecified error occur.
- */
-RCL_PUBLIC
-RCL_WARN_UNUSED
-rcl_ret_t
-rcl_system_time_now(rcl_time_point_value_t * now);
-
-/// Retrieve the current time as a rcl_time_point_value_t object.
-/**
- * This function returns the time from a monotonically increasing clock.
- * The closest equivalent would be to std::chrono::steady_clock::now();
- *
- * The resolution (e.g. nanoseconds vs microseconds) is not guaranteed.
- *
- * The now argument must point to an allocated rcl_time_point_value_t object,
- * as the result is copied into this variable.
- *
- * <hr>
- * Attribute          | Adherence
- * ------------------ | -------------
- * Allocates Memory   | No
- * Thread-Safe        | Yes
- * Uses Atomics       | No
- * Lock-Free          | Yes [1]
- * <i>[1] if `atomic_is_lock_free()` returns true for `atomic_int_least64_t`</i>
- *
- * \todo TODO(tfoote): consider moving this to rmw for more reuse
- *
- * \param[out] now a struct in which the current time is stored
- * \return `RCL_RET_OK` if the current time was successfully obtained, or
- * \return `RCL_RET_INVALID_ARGUMENT` if any arguments are invalid, or
- * \return `RCL_RET_ERROR` an unspecified error occur.
- */
-RCL_PUBLIC
-RCL_WARN_UNUSED
-rcl_ret_t
-rcl_steady_time_now(rcl_time_point_value_t * now);
```

Comment:
I don't think the functions should be aliases. First, that makes it harder to see where the functionality is coming from. Second, in this case the function is not even supposed to be used by users of `rcl` anyway. Users will use the time source / time source type instead. Therefore I would prefer to leave this as-is.